test/llvm_targets.zig

```diff
@@ -3,6 +3,8 @@ const Cases = @import("src/Cases.zig");
 
 const targets = [_]std.zig.CrossTarget{
     .{ .cpu_arch = .aarch64, .os_tag = .freestanding, .abi = .none },
+    .{ .cpu_arch = .aarch64, .os_tag = .ios, .abi = .none },
+    .{ .cpu_arch = .aarch64, .os_tag = .ios, .abi = .simulator },
```

Comment:
Could you also include `x86_64-ios-simulator` which is also a vaild target that we used to support?. In the future, we might want to expand across more Apple platforms, but given that I haven't done any checks on any other platform but `macos` and `ios`, I think it is safe to defer adding those until later.